src/ui/component/errorBoundary/index.js

```diff
@@ -0,0 +1,4 @@
+import ErrorBoundary from './view';
+
+// TODO: bring in device/user(?) info in the future
+export default ErrorBoundary;
```

Comment:
What is the intended purpose of this component?  The name gives me the impression this may be something we want to discuss.

Reply:
The purpose is a global error handler. If we don't catch anything below, catch this at the top and report it. My idea was that we could include user info if we wanted to reach out/more info on what could have caused the issue, but if we go to internal-apis we should have everything there.